src/familybot/FamilyBot.py

```
@@ -480,12 +481,72 @@ def purge_family_library_cache_cli() -> None:
+            conn.commit()
+
+            print(
+                f"✅ Price cache purge complete! Deleted {price_count} price entries and {mapping_count} mapping entries."
+            )
+            print("\n🔄 Next steps:")
+            print("- Run 'just populate-prices' to rebuild price cache")
+        else:
+            print("❌ Price cache purge cancelled.")
+    except sqlite3.Error as e:
+        print(f"❌ Error purging price cache: {e}")
+        logger.error(
```

Ruff (0.15.7)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)